src/xGadgetron/cGadgetron/gadgetron_x.cpp

```diff
@@ -208,7 +210,7 @@ AcquisitionsProcessor::process(MRAcquisitionData& acquisitions)
 	}
 
 	uint32_t na = sptr_acqs->number();
-	//std::cout << na << '\n';
+	//std::cout << na << " acquisitions processed\n";
```

Comment:
Do these lines do anything, because they are comments, right?

Reply:
@johannesmayer of course they do not, this is a debugging print, which is often needed so I keep it commented out.